packages/components/src/VirtualizedList/VirtualizedList.scss

```
@@ -25,6 +25,15 @@
 	}
 }
 
+:global(.tc-list-table [role='columnheader']) {
```

```suggestion
:global(.tc-list-table [role='columnheader']) {
```
`.tc-list-table` is global? Isn't the whole project on CSS modules? That's weird.

---

It looks like it is indeed. What do we feel is the benefit of this compound selector versus adding a classname, albeit global if needed, on the column headers? Wouldn't that be more straightforward?